README.md

```diff
@@ -71,10 +71,10 @@ For other distributions you can find the required dependencies [here.](https://l
 
 #### Go
 
-To build Lotus, you need a working installation of [Go 1.19.12 or higher](https://golang.org/dl/):
+To build Lotus, you need a working installation of [Go 1.20.10 or higher](https://golang.org/dl/):
```

Comment:
Lotus supports Go 1.19.12 so this is not necessarily correct

Reply:
@nonsense bump

Reply:
```
$ go version
go version go1.19.2 darwin/arm64
```

```
~/code/src/github.com/filecoin-project/lotus (main)
$ make clean all
Makefile:14: Your Golang version is go1.19.2
Makefile:15: *** Update Golang to version to at least 1.20.7.  Stop.
```

See: https://github.com/filecoin-project/lotus/blob/master/GO_VERSION_MIN

Reply:
Ah, you're right. I was looking at the release branch.